docs/src/adoc/index.adoc

```diff
@@ -2333,29 +2333,32 @@ methods are just creating and binding custom implementations of
 
 === JdbiConfig
 
-TODO:
-
-* ConfigRegistry is a registry of `JdbiConfig` implementations.
-* Usage:
-** ConfigRegistry.get(Class<T extends JdbiConfig>)
-** Configurable.getConfig(Class<T extends JdbiConfig>)
-** Configurable.configure(Class<T extends JdbiConfig>, Consumer<T>)
-* The `registerXyz` methods everywhere are part of the `Configurable`
-  interface, and are actually being handled under the hood by `ConfigRegistry`.
-* Jdbi, Handle, SQL objects, and SQL statements (updates, queries, etc) all
-  hold a ConfigRegistry.
-* Inheritance of configuration at time of instantiation:
-** When Jdbi creates a Handle, the handle gets a copy of the Jdbi's config
-   registry state.
-** When Jdbi creates an on-demand extension object (i.e. a SQL object), the
-   extension gets a copy of the Jdbi's config registry state.
-** When a Handle creates a SQL statement object such as a Query or Update, that
-   SQL statement gets a copy of the Handle's config registry state.
-** When a Handle creates an attached extension object (i.e. a SQL object), that
-   extension gets a copy of the Handle's config registry state.
-* When a Jdbi create an on-demand SQL object, the SQL object receives a
-  configuration at the moment it was created. The same applies when Handle
-  creates a SQL statement, or attaches a SQL Object.
+Configuration is managed by the
+link:{jdbidocs}/core/config/ConfigRegistry.html[ConfigRegistry]
+class.  Each Jdbi object that represents a distinct database context
+(for example, *Jdbi* itself, a *Handle* instance, or an attached SqlObject
+class) gets its own configuration registry.  Most contexts implement the
+link:{jdbidocs}/core/config/Configurable.html[Configurable] interface which
+allows modification of its configuration as well as retrieving the current
+context's configuration for use by Jdbi core or extensions.
+
+When a new configurable context is created, it inherits a copy of its
+parent configuration at the time of creation - further modifications will
+not affect already created configuration contexts.
```

Comment:
Maybe clarify "further modifications _on the original_ will not affect already-created copies."